packages/discord.js/src/structures/ThreadMember.js

```diff
@@ -35,12 +35,23 @@ class ThreadMember extends Base {
      */
     this.id = data.user_id;
 
-    this._patch(data);
+    this._patch(data, extra);
   }
 
-  _patch(data) {
+  _patch(data, extra = {}) {
     if ('join_timestamp' in data) this.joinedTimestamp = Date.parse(data.join_timestamp);
     if ('flags' in data) this.flags = new ThreadMemberFlagsBitField(data.flags).freeze();
+
+    if ('member' in data) {
+      /**
+       * The guild member associated with this thread member.
```

Comment:
Well this won't be confusing with the other aptly named `guildMember` getter 😢

Reply:
This one is `@private`. The getter first resolves this property before attempting to retrieve from the cache.